application/listeners/posthelpers.go

```diff
@@ -6,83 +6,166 @@ import (
 	"github.com/silinternational/wecarry-api/notifications"
 )
 
-type PostMsgRecipient struct {
-	nickname string
-	email    string
+// The intention is to allow the tests to override this with their own Send function
```

Comment:
This was the purpose of the dummy email service. We can add methods to it to query sent emails. I added a couple methods so far.